dcicutils/s3_utils.py

```diff
@@ -260,52 +271,73 @@ def health_page_get(self, health_page_key: str, *, default: Any = None, force: b
 
     ACCESS_KEYS_S3_KEY = 'access_key_admin'
 
-    def get_access_keys(self, name=ACCESS_KEYS_S3_KEY):
+    def get_access_keys(self, name: str = ACCESS_KEYS_S3_KEY) -> ServerAuthDict:
+        # TODO: Make .get_access_keys() do better error checking?
+        #       I have marked the return value as ServerAuthDict because it's CLEAR that various callers
+        #       assume that values coming back from this function are in the right form. e.g.,
+        #       ff_utils.get_authentication_with_server will give a hard error if this ever returns
+        #       something with a server missing.  So presumably we mostly do the right thing here.
```

Comment:
These access key related methods are all operating under the assumption that these S3 files follow the convention that `load_access_keys` uses ie:

https://github.com/4dn-dcic/fourfront/blob/master/src/encoded/commands/load_access_keys.py#L77-L79
https://github.com/dbmi-bgm/cgap-portal/blob/master/src/encoded/commands/load_access_keys.py#L68-L70